backend/internal/handlers/webhooks/webhooks.go

```diff
@@ -84,11 +84,35 @@ func (s *WebHookService) baseRepoInitialization(c *fiber.Ctx, pushEvent github.P
 		return errs.BadRequest(errors.New("invalid repository data"))
 	}
 
-	// Create development branch
-	_, err := s.githubappclient.CreateBranch(c.Context(), *pushEvent.Repo.Organization,
+	// Create necessary repo branches
+	_, err := s.appClient.CreateBranch(c.Context(), *pushEvent.Repo.Organization,
 		*pushEvent.Repo.Name,
 		*pushEvent.Repo.MasterBranch,
-		"dev")
+		"development")
```

Comment:
Hard coded strings + duplicated code here. I'd like a place that defines the standard list of branches created, with this looping through them

Reply:
Good point. Allow the user to specify the branches is a logged future enhancement, but will implement the looping.